configure.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
these top level yaml files should be removed, if they are example then we should just document them or place them into some sort of examples folder for documentation